Monika After Story/game/script-brbs.rpy

```diff
@@ -671,6 +671,83 @@ label monika_idle_working_callback:
         m 6ckc "..."
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_idle_screen_break",
+            prompt="I need a break from the screen.",
+            category=['be right back'],
+            pool=True,
+            unlocked=True
+        ),
+        markSeen=True
+    )
+
+label monika_idle_screen_break:
+    if mas_isMoniNormal(higher=True):
+
+        if mas_getEV('monika_idle_screen_break').timePassedSinceLastSeen_dt(mas_getSessionLength()):
+
+            if mas_getSessionLength() < datetime.timedelta(minutes=40):
+                m 1esc "Oh,{w=0.3} okay."
+                m 1dsu "You've not been here for that long but if you say you need a break, then you need a break."
+            elif mas_getSessionLength() < datetime.timedelta(minutes=150):
+                m 1eua "Going to rest your eyes for a bit?"
+            else:
+                m 1lksdla "Yeah, you probably need that, don't you?"
+
+            m 1hub "I'm glad you're taking care of your health, [player]."
+
+            if not persistent._mas_pm_works_out:
+                m 3eua "Why not take the opportunity to do a few stretches as well, hmm?"
+                m 1eub "Anyway, come back soon!~"
+            else:
+                m 1eub "Come back soon!~"
+
+        else:
+            m 1eua "Taking another break, [player]?"
+            m 1hua "Come back soon!~"
+
+    elif mas_isMoniUpset():
+        m 2esc "Oh...{w=0.5} {nw}"
+        extend 2rsc "Okay."
+
+    elif mas_isMoniDis():
+        m 6ekc "Alright."
+
+    else:
+        m 6ckc "..."
+
+    $ mas_idle_mailbox.send_idle_cb("monika_idle_screen_break_callback")
+    $ persistent._mas_idle_data["monika_idle_screen_break"] = True
+    return "idle"
+
+label monika_idle_screen_break_callback:
+    $ wb_quip = mas_brbs.get_wb_quip()
```

Comment:
You only use `wb_quip` in the normal+ flow, you can just define this in that flow so we're not getting a quip needlessly potentially.